include/glow/Backends/DeviceManager.h

```diff
@@ -29,6 +29,10 @@
 namespace glow {
 namespace runtime {
 
+/// Callback signalling success/failure of evicting a function from a Device.
+using EvictCBTy =
```

Comment:
Nit: Does the name`EvictCBTy` need to be more descriptive? E.g. if we could potentially evict something else besides functions, we should better call it `EvictFunctionCBTy`.